wire-ios/Wire-iOS/Sources/UserInterface/StartUI/Common/EmptySearchResultsView.swift

```diff
@@ -82,37 +82,48 @@ final class EmptySearchResultsView: UIView {
     private let isSelfUserAdmin: Bool
     private let isFederationEnabled: Bool
 
-    private let stackView: UIStackView
+    /// Contains the `stackView`.
+    private let scrollView = UIScrollView()
+    private let stackView = UIStackView()
     private let iconView     = UIImageView()
     private let statusLabel  = DynamicFontLabel(fontSpec: .normalRegularFont,
                                                 color: LabelColors.textSettingsPasswordPlaceholder)
```

Comment:
Maybe the `=` characters should be aligned with in lines?

Reply:
done